rtublen/Studentportalen
# Features/1008 add profile page

Adds profile page for Frans

## How to verify change

1. Go to studentportalen.website.
2. Check for frans.cshtml under profiles.
